replay_analysis/analysis/stats/stats_manager.py

```diff
@@ -0,0 +1,55 @@
+from typing import Dict, List
+
+from ...analysis.stats.stats_list import StatsList
+from ...generated.api import game_pb2
+from ...generated.api.player_pb2 import Player
+from ...generated.api.stats.player_stats_pb2 import PlayerStats
+from ...generated.api.stats.team_stats_pb2 import TeamStats
+from ...generated.api.team_pb2 import Team
+from ...json_parser.game import Game
+
+
+class StatsManager:
+
+    def get_stats(self, game: Game, proto_game: game_pb2.Game, player_map: Dict[str, Player],
+                  data_frames):
+        self.calculate_player_stats(game, proto_game, player_map, data_frames)

```

Comment:
Would be good to specify this order in the image - important as as the stats calculated before can't depend on those calculated after.
Also rearrange the function definitions in this file to follow the same order :P